ci/Jenkinsfile.update_env

```diff
@@ -0,0 +1,45 @@
+def gh_branch = 'feature/jenkinsfile'
+def gh_repo = 'github.com/status-im/status-im.github.io.git'
+def gh_user = 'status-im-auto'
+def gh_email = 'auto@status.im'
+
+node('linux') {
+  stage('Git Prep') {
+    /* WARNING: Job has to use the "Check out to specific local branch" */
+    checkout scm
+    sh "git config user.name ${gh_user}"
+    sh "git config user.email ${gh_email}"
+    sh "git config --global push.default simple"
+  }
+
+  stage('Update URLs') {
+    def changes = 0
+    println "params: ${params}"
+    for (param in ['APK_URL', 'IOS_URL', 'DMG_URL', 'OSX_URL']) {
+      println "param: ${param} value: ${params.get(param)}"
+      if (!params.get(param)) {
+        continue /* skip if prameter was not provided */
+      }
+      sh """
+        sed -i -e \\
+          's#env.${param}=.*#env.${param}=\"${params.get(param)}\"#g' \\
```

Comment:
It looks like it's duplicated in `update_env.sh`. Would it be possible to avoid that?

Reply:
Oh yeah, I forgot to remove that script!

Reply:
removed